mne/viz/raw.py

```diff
@@ -576,18 +585,27 @@ def plot_raw_psd(raw, tmin=0., tmax=np.inf, fmin=0, fmax=np.inf, proj=False,
     fig : instance of matplotlib figure
         Figure with frequency spectra of the data channels.
     """
+    if average and spatial_colors:
+        raise ValueError('Average and spatial_colors cannot be enabled '
+                         'simultaneously.')
+    elif spatial_colors is None:
+        # XXX: deprecation
+        warn('In version 0.15 average will default to False and '
+             'spatial_colors to True.')
```

Comment:
don't emit this warning if `average=False`, there is no need (no deprecation cycle) because the default is and will stay `True`